### PR TITLE
My First Pull Request

### PR DESCRIPTION
**Pull request by andreglenn-ipreo**
Changes to AG readme file 😀

### CODE REVIEW

**Comment by andreglenn-ipreo** [comment]
test

**Review by andreglenn-ipreo — COMMENTED** [review]
review comment goes here

File: _posts/0000-01-02-andreglenn-ipreo.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![fresh-prince](https://cloud.githubusercontent.com/assets/16547949/25400882/f44a7464-29c1-11e7-8f45-327d67cb961a.jpg)
 {: .center}
 
-Caption here
+Hey, its not my fault that Carlton is devoid of Funk
```
Comment (githubteacher) [inline _posts/0000-01-02-andreglenn-ipreo.md]: 👍

**Review by githubteacher — APPROVED** [review]
💯

**Pull request merged**